### ISSUE TITLE
False negative in color-function-notation

### ISSUE THREAD

**Issue by aarongoldenthal** [stylelint/stylelint#5704]
### What steps are needed to reproduce the bug?

The `color-function-notation` rule set to `modern` does not find an error with `rgba` notation missing the `/`, e,g, `rgb(255 255 255 10%)` does not raise an error, although it should be `rgb(255 255 255 / 10%)`.

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "color-function-notation": "modern"
  }
}
```

### How did you run Stylelint?

CLI with `stylelint "**/*.css"` (or other applicable extensions)

### Which version of Stylelint are you using?

`14.0.1`

### What did you expect to happen?

Rule should indicate an error.

### What actually happened?

No errors were reported.

**Comment by jeddy3** [comment]
@aarongoldenthal Thanks for the report.

The following isn't valid CSS:

```css
a { color: rgb(255 255 255 10%); }
```

You can catch validation errors using a validator like [CSSTree](https://csstree.github.io/docs/validator.html) which is [available as a Stylelint plugin](https://github.com/csstree/stylelint-validator).

See https://stylelint.io/about/linting#validators.

In the future, we may change our parser to one that supports validation but in the meantime the recommendation is to use a dedicated validator.
